sphinx_rtd_theme/layout.html

```diff
@@ -171,9 +171,11 @@
            <div itemprop="articleBody">
             {% block body %}{% endblock %}
            </div>
+           {% if self.comments()|trim %}
```

Comment:
Why trim?

Reply:
Without trimming, the `comment` block will be printed if it is not exactly empty but contains newlines or spaces. I don't know if that is likely to happen, but I thought it would be cheap enough to call `trim`